File: crates/worldgen/src/multimodal_knowledge_representation_workflow_fabric.rs
//! Worldgen P04-F14 multimodal knowledge-representation workflow fabric.
use super::knowledge_workflow_support::{self, KnowledgeWorkflowRequest, KnowledgeWorkflowReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P04-F14"; pub const CONTRACT_VERSION:&str="worldgen-multimodal-knowledge-workflow/1.0";
pub fn worldgen_multimodal_knowledge_representation_workflow_fabric_manifest()->serde_json::Value{knowledge_workflow_support::manifest(FEATURE_ID,CONTRACT_VERSION,"KnowledgeWorkflowRequest1@1","multimodal multi-study","A2")}
pub fn schedule_worldgen_multimodal_knowledge_representation_workflow(request:&KnowledgeWorkflowRequest)->Result<KnowledgeWorkflowReceipt,knowledge_workflow_support::KnowledgeWorkflowError>{knowledge_workflow_support::schedule(request,FEATURE_ID,CONTRACT_VERSION,"multimodal multi-study",true,false)}
pub use knowledge_workflow_support::{KnowledgeWorkflowError,KnowledgeWorkflowReceipt as WorldgenMultimodalKnowledgeWorkflowReceipt,KnowledgeWorkflowRequest as WorldgenMultimodalKnowledgeWorkflowRequest};

File: python/prism_sdk/multimodal_knowledge_representation_workflow_fabric.py
"""Worldgen P04 AFA-worldgen-P04-F14 product wrapper."""
from .worldgen_knowledge_workflow_support import KnowledgeWorkflowRequest, KnowledgeWorkflowReceipt, manifest as _manifest, schedule as _run
FEATURE_ID="AFA-worldgen-P04-F14"
CONTRACT_VERSION="worldgen-multimodal-knowledge-workflow/1.0"
def worldgen_multimodal_knowledge_representation_workflow_fabric_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="KnowledgeWorkflowRequest1@1", scale="multimodal multi-study", autonomy_tier="A1")
def schedule_worldgen_multimodal_knowledge_representation_workflow(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="multimodal multi-study", require_approval=False, require_federation=False)
__all__=["KnowledgeWorkflowRequest","KnowledgeWorkflowReceipt","worldgen_multimodal_knowledge_representation_workflow_fabric_manifest","schedule_worldgen_multimodal_knowledge_representation_workflow"]
